src/ResourceManager/Sql/Commands.Sql/help/Clear-AzSqlDatabaseVulnerabilityAssessmentRuleBaseline.md

```diff
@@ -20,7 +20,7 @@ Clear-AzSqlDatabaseVulnerabilityAssessmentRuleBaseline [-InputObject <AzureSqlDa
 
 ## DESCRIPTION
 The **Clear-AzSqlDatabaseVulnerabilityAssessmentRuleBaseline** cmdlet removes the vulnerability assessment rule baseline.
-Note that you need to run *Enable-AzSqlServerAdvancedThreatProtection* and *Set-AzSqlDatabaseVulnerabilityAssessmentSettings* cmdlet as a prerequisite for using this cmdlets.
+Note that you need to run *Enable-AzSqlServerAdvancedThreatProtection* and *Update-AzSqlDatabaseVulnerabilityAssessmentSettings* cmdlet as a prerequisite for using this cmdlets.
```

Comment:
Does `Set-AzSqlDatabaseVulnerabilityAssessmentSettings` still exist? Because we cannot remove it outside of a breaking change release. And, I didn't see a cmdlet alias on `Update-AzSqlDatabaseVulnerabilityAssessmentSettings`

Reply:
Set-AzSqlDatabaseVulnerabilityAssessmentSettings never existed. It's an error in the documentation that I noticed and fixed